### PR TITLE
adding a CSS layout for a good website

### LINKED ISSUE #24: Add Styling
I would like to add styling to make this web page more attractive

### PR DESCRIPTION
**Pull request by apoodwivedi**
ADDED A CSS LAYOUT
fix #24

### CODE REVIEW

**Comment by CodeAsherPlayz** [comment]
Closes #24

File: index.html [hunk 1]
```diff
@@ -6,6 +6,104 @@
 
     <!-- Set the title of your site here -->
     <title> HTML Patterns </title>
+    <title>CSS Template</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (CodeAsherPlayz) [inline index.html:11]: Please indent this to the title. Thanks!

**Review by CodeAsherPlayz — CHANGES_REQUESTED** [review]
Just needs to fix Line 10 and 11.

**Comment by CodeAsherPlayz** [comment]
Link #54 here.

**Comment by mrmrs** [comment]
Thanks for the pull request but I am not looking to add any styles to this page. The point is to have unstyled HTML blocks that you can have as a jumping off point. Feel free to maintain your own fork with default styles.

**Pull request closed without merging**